subsys/zigbee/Kconfig

```diff
@@ -171,6 +171,31 @@ endmenu #menu "Zigbee Log configuration"
 
 menu "ZBOSS osif configuration"
 
+menuconfig ZIGBEE_HAVE_SERIAL
+	bool "Enable UART serial interface for ZBOSS CLI"
+	select SERIAL
+	select UART_ASYNC_API
+
+if ZIGBEE_HAVE_SERIAL
+
+config ZIGBEE_UART_DEVICE_NAME
+	string "Zigbee UART device name"
+	default "UART_1"
+
+config ZIGBEE_UART_TX_TIMEOUT
+	int "Timeout value for starting asynchronous transmission, in milliseconds"
+	default -1
```

Comment:
What does `-1` mean for the timeout? It does not look like a valid parameter for the `K_MSEC` macro where it is also passed. Also using it directly in the call to `uart_tx` looks fragile. What if `SYS_FOREVER_MS` definition is changed in some future?

Reply:
Honestly, I do not see a way to set a timeout value to "forever" through Kconfig. I will update it to the non-zero value, but I still think that a Kconfig macro for the infinity symbol value would be useful in some cases.

Reply:
Proposition included in the PR.